.github/workflows/statuses.yml

```diff
@@ -0,0 +1,23 @@
+name: Docs Build
+
+on:
+  push:
+    branches:
+      - master
+      - st-statuses-action
```

Comment:
This was for testing but since I'm on a fork, this doesn't work anyway, it seems.

Reply:
```suggestion
```